### PR TITLE
Refactor VotingMachine Redeem

### PR DESCRIPTION
**Pull request by AugustoL**

### CODE REVIEW

File: contracts/dao/votingMachine/VotingMachine.sol [hunk 1]
```diff
@@ -1510,4 +1378,11 @@ contract VotingMachine {
     function getInactiveProposalsCount(address _avatar) public view returns (uint256 inactiveProposalsCount) {
         return inactiveProposals[_avatar].length();
     }
+
+    /**
+     * @dev Helper function used in test to execute a real math lib multiplication
+     */
+    function multiplyRealMath(uint256 a, uint256 b) public pure returns (uint256) {
```
Comment (MiltonTulli) [inline contracts/dao/votingMachine/VotingMachine.sol:1385]: Question: What is the idea behind this?

File: contracts/utils/RealMath.sol [hunk 2]
```diff
@@ -64,7 +64,7 @@ library RealMath {
     /**
      * Multiply one real by another. Truncates overflows.
      */
-    function mul(uint256 realA, uint256 realB) private pure returns (uint256) {
+    function mul(uint256 realA, uint256 realB) internal pure returns (uint256) {
```
Comment (MiltonTulli) [inline contracts/utils/RealMath.sol:67]: 👍

**Review by MiltonTulli — APPROVED** [review]
Looks good to me! 
You can `yarn docify` to update votinmachine doc or I can do it later.

File: contracts/dao/votingMachine/VotingMachine.sol (hunk 1 above)
Comment (AugustoL) [reply contracts/dao/votingMachine/VotingMachine.sol:1385]: Just for tetsing, I use it to do off chain calculations for score/threshold comparisons.

**Pull request merged**